README.md

```diff
@@ -22,7 +22,9 @@ For each language that the tool supports, the libyear metric is computed for eac
 
 ### Source Code Repositories
 
-* Git
+* [github.com/corgibytes/freshli](https://github.com/corgibytes/freshli)
+* [github.com/fslaborg/XPlot](https://github.com/fslaborg/XPlot)
+  * Freshli currently pulls in XPlot as a submodule as the current release does not include a critical bug fix
```

Comment:
This section was meant to list the source code repositories that were supported for history. I'm going to move the content that you added to somewhere else.

Reply:
You've actually got the important bits elsewhere, so I'm going to remove this and add more context here instead.

Reply:
Ah, gotcha. I wasn't quite sure and assumed... 😬